pkgs/applications/misc/todoist-electron/default.nix

```diff
@@ -4,18 +4,18 @@
 
 stdenv.mkDerivation rec {
   pname = "todoist-electron";
-  version = "1.19";
+  version = "1.20";
 
   src = fetchurl {
     url = "https://github.com/KryDos/todoist-linux/releases/download/${version}/Todoist_${version}.0_amd64.deb";
-    sha256 = "1w0l7k7wmbhwzv71cffsir0q7zg9m0617fmyvd4a01b6flpxrpfx";
+    sha256 = "0w885xqy1304cp6b0jll5lvm6b1zd1ciqjl97d2hkdi8c9gv3bqx";
   };
 
   desktopItem = makeDesktopItem {
     name = "Todoist";
     exec = "todoist";
     desktopName = "Todoist";
-    categories = "Utility;Productivity";
```

Comment:
Is this category not appropriate?

Reply:
The desktop file validation didn't recognize "Productivity" as a valid value for some reason (even though it did last time?), so I just took it out.

```
these derivations will be built:
  /nix/store/6l1xplwrlmb9bdljny0c1ghqrbzni0ab-Todoist.desktop.drv
  /nix/store/zvpg6395hqr6r4f4gdbk1gi3xxadjd1a-todoist-electron-1.20.drv
building '/nix/store/6l1xplwrlmb9bdljny0c1ghqrbzni0ab-Todoist.desktop.drv'...
Running desktop-file validation
/nix/store/p86gdc6hb8hhjysx1wyh1aysw4ilhcrw-Todoist.desktop/share/applications/Todoist.desktop: error: value "Utility;Productivity" for key "Categories" in group "Desktop Entry" contains an unregistered value "Productivity"; values extending the format should start with "X-"
builder for '/nix/store/6l1xplwrlmb9bdljny0c1ghqrbzni0ab-Todoist.desktop.drv' failed with exit code 1
cannot build derivation '/nix/store/zvpg6395hqr6r4f4gdbk1gi3xxadjd1a-todoist-electron-1.20.drv': 1 dependencies couldn't be built
error: build of '/nix/store/zvpg6395hqr6r4f4gdbk1gi3xxadjd1a-todoist-electron-1.20.drv' failed
```

Should I have "X-Productivity" instead?